src/python/grpcio/grpc/experimental/aio/_call.py

```diff
@@ -146,31 +147,49 @@ def _create_rpc_error(initial_metadata: Optional[MetadataType],
 
 
 class Call(_base_call.Call):
+    """Base implementation of client RPC Call object.
+
+    Implements logic around final status, metadata and cancellation.
+    """
     _loop: asyncio.AbstractEventLoop
     _code: grpc.StatusCode
     _status: Awaitable[cygrpc.AioRpcStatus]
     _initial_metadata: Awaitable[MetadataType]
     _locally_cancelled: bool
+    _cython_call: cygrpc._AioCall
 
-    def __init__(self) -> None:
+    def __init__(self, deadline: Optional[float], channel: cygrpc.AioChannel,
```

Comment:
You're right. I did miss that. The docstring says "A bool indicates if the cancellation is performed or not." That's not quite what I'm seeing in the code here though. As currently written, I would assume that `False` means cancellation failed. What I'm seeing here is a `True` value means that this particular invocation resulted in active cancellation of the RPC, whereas a value of `False` implies that no work was performed to cancel this RPC as it was already cancelled.

I'm now wondering what the use case for the return value of this function is. Perhaps we should just document that it's idempotent and return no value at all. In case there is an actual problem cancelling the RPC, we can raise an exception.

Reply:
Discussed offline. Basically, stamp coupling.